wa-public/src/app/pages/success/success.component.ts

```diff
@@ -15,7 +15,7 @@ import { take, mergeMap } from 'rxjs/operators';
         <ion-title> {{ title }}</ion-title>
 
         <ion-buttons slot="primary">
-          <ion-button (click)="actionSvc.logout()">
+          <ion-button (click)="logout()">
             <ion-label>Logout</ion-label>
             <ion-icon name="log-out"></ion-icon>
           </ion-button>
```

Comment:
Nitpick/suggestion: since this is being displayed in basically every page, we may want to move it to a separate header component that we can just put on the page, reducing code duplication.

Reply:
Yes.. was done this way initially for development speed. To centralize it I'll need to create a more central state. This can be done in the future.